CODEOWNERS

```diff
@@ -9,6 +9,13 @@ homeassistant/*.py @home-assistant/core
 homeassistant/helpers/* @home-assistant/core
 homeassistant/util/* @home-assistant/core
 
+
+
+
+
+
+
+
```

Comment:
Is all the blank lines intended?

Reply:
oh lol, no, should not have committed. That was a test.

Reply:
I have a new thing where we don't generate if validation failed.

Reply:
I am thinking about allowing the current get_component references, but not allowing new ones, while we figure out how to deal with calls to `hass.components`. Thinking about introducing a new "soft" dependency, integrations you want available, but don't need set up. Does that make sense?

Reply:
Yes! I definitely think we need that soft dependency. Eg some components, like owntracks, mysensors, need mqtt only in some cases.

Reply:
https://github.com/home-assistant/architecture/issues/196